src/unix/lwt_engine.ml

```diff
@@ -197,6 +197,57 @@ end
 
 class libev_deprecated = libev ()
 
+let unix_fd_to_fd (fd: Unix.file_descr) : int = Obj.magic fd
+
+class libuv () = object
+  inherit abstract
+
+  val loop = Luv.Loop.default ()
+
+  method loop = loop
+
+  method private cleanup = Luv.Loop.stop loop
+
+  method iter block =
+    try
+      match (block) with
+      | true -> Luv.Loop.run ~loop ~mode:`ONCE () |> ignore
+      | false -> Luv.Loop.run ~loop ~mode:`NOWAIT () |> ignore
+    with exn ->
+      Luv.Loop.stop loop;
```

Comment:
I would omit this, as we are now using the default loop. I guess we don't need the exception handler at all here, then.

Reply:
Also, IIRC, `Luv.Loop.run` can't raise any exceptions, so another argument in favor of not trying to handle any here.